### ISSUE TITLE
Exit code 0xC0000005 importing matplotlib.py

### ISSUE THREAD

**Issue by sjsy** [matplotlib/matplotlib#11552]
### Bug report

**Bug summary**

using the following line in my code leads to  

> Process finished with exit code -1073741819 (0xC0000005)
`import matplotlib.pyplot as plt` 

**Code for reproduction**

`import matplotlib.pyplot as plt`

**Matplotlib version**
  * Operating system: Windows 10 64bit
  * Matplotlib version: 2.2.2
  * Matplotlib backend (`print(matplotlib.get_backend())`): Qt5Agg
  * Python version: Python 3.6.6 |Anaconda, Inc.| (default, Jun 28 2018, 11:27:44) [MSC v.1900 64 bit (AMD64)] on win32
  * Jupyter version (if applicable):
  * Other libraries: tensorflow, keras, imageio, scikit-image.

I installed matplotlib using conda. I used default channel.

**Comment by tacaswell** [comment]
suspect related to https://github.com/matplotlib/matplotlib/issues/3316

**Comment by sulaimanvesal** [comment]
Solved the problem using the following code. I don't know why this happens with Pycharm in Windows 10 but calling matplotlib.use('Agg') solved the issue.
```
import matplotlib
matplotlib.use('Agg')
import matplotlib.pyplot as plt
```

**Comment by timhoffm** [comment]
Can this be closed or is any action on the matplotlib side required?

**Comment by tacaswell** [comment]
It is likely due to crossed / broken installations, I think can close this.  @sulaimanvesal If you have further questions about installation can you create a topic in https://discourse.matplotlib.org/c/community/install?

**Comment by xaviergoby** [comment]
@sulaimanvesal Thank you sharing/mentioning this solution method, it worked for me! However, I am still completely in the dark as to what is the cause for this issue. In fact this issue truly popped out of the blue so suddenly whilst I was coding in my PyCharm project that I could not tell what could have possibly been the culprit for it. After an hour's long hunt for a solution both on the internet and wrangling my way through all my code and my projects env it was only until after I finally decided to slowly but calmly riddle my scripts with print statements (yes, I know, awfully foolish practice on my part but for Christ sake had I just had absolutely enough of the darn bug/issue) that I managed to pinpoint the 

`import matplotlib.pyplot as plt `

line of code as having been the bain of my existence for the last 1 to 2 hours.
If someone happens to have a better understanding on this issue/bug/error (something a little than simply an opinion/guess if you don't mind please) then I would greatly appreciate some insight please :).

Oh and yes of course, before I forget, this is what the output/message which my console would return to me when I was importing matplotlib as I mentioned above (before implementing the fix mentioned by @sulaimanvesal):

Process finished with exit code -1073741819 (0xC0000005)

**Comment by Lucky-Light-Sun** [comment]
when I use the code bellow, my pycharm finished with exit code -1073741819 (0xC0000005). This really drive me crazy for all day!
```python
import matplotlib.pyplot as plt
plt.plot(range(0, 10, 1))
plt.show()
```
![image](https://user-images.githubusercontent.com/60919766/140514096-f82a49c6-022f-4da4-aaa1-6211410afbaa.png)

But if I comment out the last line of code, process finished with exit code 0.
```python
import matplotlib.pyplot as plt
plt.plot(range(0, 10, 1))
# plt.show()
```
![image](https://user-images.githubusercontent.com/60919766/140514790-82f5ec9c-26dc-4422-945f-5fc33d1ca870.png)

But I want to draw a chart, how could I do?

  My os is Windows 10 Proffesional
  python 3.8.12
  matplotlib 3.1.3

By the way, execute this python file in cmd, but it still does't word!
What's more, I use conda virtual enviroment and execute it with python 3.7.11, python 3.9.7, python 3.6.13. The result disappointed me
But when I used Python 3.5.6,  the chart came out!
![image](https://user-images.githubusercontent.com/60919766/140515667-9f49e699-f53d-4429-afc2-a8fbc97a5889.png)

I really really want to know why I can't plot charts with higher python version?
Wish your response!

**Comment by tacaswell** [comment]
@Lucky-Light-Sun I suspect that you are have an issue with your local installation rather than a bug in Matplotlib.

https://answers.microsoft.com/en-us/windows/forum/all/application-error-0xc0000005/6224ae45-a251-4f21-b076-74524618d00a?auth=1 suggests that this could be a permissions issue.  I suggest you create a new conda environment and ensure that all of the files are owned by you.

If the `show` is what is causing the exit that strongly suggests that this is related to the GUI toolkit, please check which backend you are using and ensure that it's dependencies are correctly installed.

**Comment by bigzcoder** [comment]
@Lucky-Light-Sun Hi , I meet a same question.But I have fixed this bug.

Firstly, I open Event Viewer, and look at the error in it.

> You should go to this dir  `Control Panel\All Control Panel Items\Administrative Tools`

![2021-12-02_141735](https://user-images.githubusercontent.com/21018496/144368481-207c6d0f-c9e7-42e6-995d-46df5d1c60d6.jpg)

then,I found that this has something to do with the version dependency of my freetype.

Secondly, So I change my freetype version

Finally. I solved the problem

![2021-12-02_142351](https://user-images.githubusercontent.com/21018496/144369279-fd06af7d-6ca9-4315-a433-05c75e3949a2.jpg)
